app/src/main/java/net/skyscanner/backpack/demo/stories/Story.kt

```diff
@@ -12,7 +12,9 @@ open class Story : ComponentDetailFragment() {
     val layoutId = arguments?.getInt(LAYOUT_ID) ?: savedInstanceState?.getInt(LAYOUT_ID)
     if (layoutId != null) {
       return inflater.inflate(layoutId, container, false).apply {
-        layoutDirection = if (arguments?.getBoolean(RTL) == true) View.LAYOUT_DIRECTION_RTL else View.LAYOUT_DIRECTION_LTR
+        if (arguments?.getBoolean(RTL) == true) {
```

Comment:
It seems to have an extra empty line here

Reply:
Just visual glitch here, the if is on line 15 and the next line is 16, so no empty line